web/milieuthemas/datapunt_api/templates/rest_framework/base.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<html lang="nl">
+{% extends "rest_framework/base.html" %}
```

Comment:
@cvriel werkt dit? In Django kan je een `{% include %}` overal zetten, maar `{% extends %}` moet bovenaan staan omdat het de hele rendering van de template overneemt, en alleen blokken merged. Alle overige tekst wordt dan genegeerd. Als dit al werkt, zal je waarschijnlijk een dubbele `<html>` in de source vinden.

Reply:
Ik zal kijken naar een andere oplossing, je hebt gelijk leidt to dubbele <html> tags...